distributed/executor.py

```diff
@@ -9,6 +9,8 @@
 import os
 from time import sleep
 import uuid
+from distributed.compatibility import Queue as pyQueue
```

Comment:
should use relative import here.  There might be a compatibility import in this file already actually.